packages/docs/src/pages/en/features/icon-fonts.md

```diff
@@ -70,68 +116,72 @@ $ npm install @mdi/font -D
 // src/plugins/vuetify.js
 
 import '@mdi/font/css/materialdesignicons.css' // Ensure you are using css-loader
-import Vue from 'vue'
-import Vuetify from 'vuetify/lib'
-
-Vue.use(Vuetify)
+import { createVuetify } from 'vuetify/lib'
 
-export default new Vuetify({
+export default createVuetify({
   icons: {
-    iconfont: 'mdi', // default - only for display purposes
+    defaultSet: 'mdi', // This is already the default value - only for display purposes
   },
 })
 ```
+<alert type="error">
+
+  **DO NOT** use a CDN link without specifying a package *version*. Failure to do so can result in unexpected changes to your application with new releases.
+
+</alert>
+
+Use this tool to search for any Material Design Icons and copy them to your clipboard by clicking the item.
+
+<icon-list />
 
 ### Material Design Icons - JS SVG
 
-Use the SVG paths as designated in [@mdi/js](https://www.npmjs.com/package/@mdi/js). This is the recommended installation when optimizing your application for production. You **ONLY** need to include this if you plan on using more than the default icons.
+Use the SVG paths as designated in [@mdi/js](https://www.npmjs.com/package/@mdi/js). This is the recommended installation when optimizing your application for production.
 
 ```bash
 $ yarn add @mdi/js -D
 // OR
 $ npm install @mdi/js -D
 ```
 
-Specify the **mdiSvg** iconfont:
-
 ```js
 // src/plugins/vuetify.js
 
-import Vue from 'vue'
-import Vuetify from 'vuetify/lib'
-
-Vue.use(Vuetify)
+import { createVuetify } from 'vuetify/lib'
+import { aliases, mdi } from 'vuetify/lib/iconsets/mdi-svg'
 
-export default new Vuetify({
+export default createVuetify({
   icons: {
-    iconfont: 'mdiSvg',
+    iconfont: 'mdi',
```

Comment:
This doesn't actually allow for arbitrary icons to be used, you still have to import anything not in the preset.

Reply:
And that is what's shown in the next snippet. Or do you want some text to be extra clear?